fe/fe-core/src/main/java/org/apache/doris/nereids/stats/JoinEstimation.java

```diff
@@ -52,37 +55,105 @@ private static Statistics estimateInnerJoin(Statistics crossJoinStats, List<Expr
         for (int i = 0; i < sortedJoinConditions.size(); i++) {
             sel *= Math.pow(sortedJoinConditions.get(i).second, 1 / Math.pow(2, i));
         }
-        return crossJoinStats.updateRowCountOnly(crossJoinStats.getRowCount() * sel);
+        Statistics innerJoinStats = crossJoinStats.updateRowCountOnly(crossJoinStats.getRowCount() * sel);
+
+        if (!join.getOtherJoinConjuncts().isEmpty()) {
+            FilterEstimation filterEstimation = new FilterEstimation();
+            innerJoinStats = filterEstimation.estimate(
+                    ExpressionUtils.and(join.getOtherJoinConjuncts()), innerJoinStats);
+        }
+        return innerJoinStats;
     }
 
     private static double estimateJoinConditionSel(Statistics crossJoinStats, Expression joinCond) {
         Statistics statistics = new FilterEstimation().estimate(joinCond, crossJoinStats);
         return statistics.getRowCount() / crossJoinStats.getRowCount();
     }
 
+    private static double adjustSemiOrAntiByOtherJoinConditions(Join join) {
+        final double non_equal_ratio = 0.5;
```

Comment:
We use camel case naming style

Reply:
Besides, similar variable has been defined in FilterEstimation with public static modifier

Reply:
Ok,  non_equal_ratio  is replaced by FilterEstimation.DEFAULT_INEQUALITY_COEFFICIENT.